plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -678,6 +683,28 @@ nothrow:
         }
     }
 
+    void opCall(ref IncreaseFilePrio data) {
+        try {
+            spinSql!(() @trusted {
+                string fileContent = readText(conf.metadataPath);
+                auto jContent = parseJSON(fileContent);
+                auto objectData = jContent.object["file-prio"];
+                Path[] prioFiles;
+                foreach (prioFileElem; objectData.array) {
+                    string strData = prioFileElem.toString;
+                    strData = strData.replace("\\", "");
```

Comment:
this replace may be unnecessary if you instead call `parseJSON` with the option `JSONOption.doNotEscapeSlashes`

Reply:
doesn't seem to work with this solution. The slashes still gets escaped